CHANGELOG.md

```diff
@@ -5,11 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.2.0
+
+### Changed
+
+- **Major breaking change:** Restructured `Plugin` API to make it thread safe ([#65](https://github.com/RustAudio/vst-rs/pull/65))
+- Fixed a number of unsoundness issues in the `Outputs` API ([#67](https://github.com/RustAudio/vst-rs/pull/67), [#108](https://github.com/RustAudio/vst-rs/pull/108))
+- Set parameters to be automatable by default ([#99](https://github.com/RustAudio/vst-rs/pull/99))
+- Updated README.md to reflect org and chat changes
```

Comment:
The `README.md` changes could probably be removed from the changelog, but I don't feel too strongly either way. LGTM.

Reply:
Yes, I also had that thought. The changelog should mention the user-visible changes to the code, rather that organizational changes.

Reply:
I reformulated it to mention the actual changes (new organization and name).